### ISSUE TITLE
[@tanstack/vue-query] Generated hooks have wrong typings

### ISSUE THREAD

**Issue by juni0r** [zenstackhq/zenstack#1056]
The generated hooks for `@tanstack/vue-query@5.24.1` have some wrong typings.

This is probably related to #945?

**Environment:**

-   ZenStack version: 1.9.0
-   Prisma version: 5.10.2
-   Database type: Postgresql

`schema.zmodel`
```prisma
datasource db {
  provider = "postgresql"
  url = env("DATABASE_URL")
}

plugin prisma {
  provider = '@core/prisma'
  format = true
}

plugin hooks {
  provider = '@zenstackhq/tanstack-query'
  target = 'vue'
  version = 'v5'
  output = 'hooks'
}

generator client {
  provider = "prisma-client-js"
}

```
**Screenshots**
<img width="657" alt="Screenshot 2024-02-25 at 21 56 23" src="https://github.com/zenstackhq/zenstack/assets/11867/29841e05-56fc-4595-b560-047da7186aa1">
<img width="667" alt="Screenshot 2024-02-25 at 21 53 19" src="https://github.com/zenstackhq/zenstack/assets/11867/6c0c150e-a33e-4e1c-9a57-7c8bc8ddff8b">
<img width="1093" alt="Screenshot 2024-02-25 at 21 49 00" src="https://github.com/zenstackhq/zenstack/assets/11867/6ed194b8-67cb-4868-87c5-8cc33c59afad">

**Comment by ymc9** [comment]
Hi @juni0r , it's already fixed with #1009 and will be included in the upcoming 1.10.0 release.

**Comment by ymc9** [comment]
Fixed in 1.10.0
